esphome/components/modbus_controller/modbus_controller.cpp

```diff
@@ -24,15 +24,22 @@ bool ModbusController::send_next_command_() {
   if ((last_send > this->command_throttle_) && !waiting_for_response() && !command_queue_.empty()) {
     auto &command = command_queue_.front();
 
-    ESP_LOGV(TAG, "Sending next modbus command to device %d register 0x%02X count %d", this->address_,
-             command->register_address, command->register_count);
-    command->send();
-    this->last_command_timestamp_ = millis();
-    // remove from queue if no handler is defined or command was sent too often
-    if (!command->on_data_func || command->send_countdown < 1) {
-      ESP_LOGD(TAG, "Modbus command to device=%d register=0x%02X countdown=%d removed from queue after send",
-               this->address_, command->register_address, command->send_countdown);
+    // remove from queue if command was sent too often
```

Comment:
Basically I'm fine with this change, but in my experience small, focused PR are preferred by the maintainers.